[esx_addons]/esx_vehicleshop/server/main.lua

```diff
@@ -25,26 +25,7 @@ end)
 
 function SQLVehiclesAndCategories()
 	categories = MySQL.query.await('SELECT * FROM vehicle_categories')
-	vehicles = MySQL.query.await('SELECT * FROM vehicles')
-
-	GetVehiclesAndCategories(categories, vehicles)
-end
-
-function GetVehiclesAndCategories(categories, vehicles)
-	for i = 1, #vehicles do
-		local vehicle = vehicles[i]
-		for j = 1, #categories do
-			local category = categories[j]
-			if category.name == vehicle.category then
-				vehicle.categoryLabel = category.label
-				break
-			end
-		end
-	end
-
-	-- send information after db has loaded, making sure everyone gets vehicle information
-	TriggerClientEvent('esx_vehicleshop:sendCategories', -1, categories)
-	TriggerClientEvent('esx_vehicleshop:sendVehicles', -1, vehicles)
+	vehicles = MySQL.query.await('SELECT vehicles.*, vehicle_categories.label AS categoryLabel FROM vehicles, vehicle_categories WHERE vehicles.category = vehicle_categories.name')
```

Comment:
Why not start by using actual joins in queries ?
`SELECT vehicles.*, vehicle_categories.label AS categoryLabel FROM vehicles JOIN vehicle_categories ON vehicles.category = vehicle_categories.name`

Here are some guides:
JOIN data in ON
Filter data in WHERE
Write explicit JOINs to make your Query more readable
Filter data in the WHERE clause instead of the JOIN to ensure it is correct and readable
Different SQL languages may have different query plans based on filtering in the ON clause vs the WHERE clause, so test the performance on your database

Reply:
You're right. I replaced.
I didn’t notice much difference in performance.

Reply:
There is no performance issues. It is just that a WHERE is for filtering data.
In SQL it is always best practice to use explicit joins and it is more readable.

Reply:
I understand. I’m not the best at more complex SQL queries yet but I’m trying. Thanks!